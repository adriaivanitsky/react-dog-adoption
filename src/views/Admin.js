import React from 'react';
import Header from '../components/Header';
import Form from '../components/Form';

export default function Admin() {
  return (
    <>
      <Header />
      <Form />
    </>
  );
}
