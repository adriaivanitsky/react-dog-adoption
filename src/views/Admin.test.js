import { render, screen } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import Admin from './Admin.js';

test('admin should render a button', () => {
  const { container } = render(
    <MemoryRouter>
      <Admin />
    </MemoryRouter>
  );
  screen.getByText('save');
  expect(container).toMatchSnapshot();
});
