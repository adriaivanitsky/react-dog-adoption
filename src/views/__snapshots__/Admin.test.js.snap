// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`admin should render a button 1`] = `
<div>
  <header>
    <a
      aria-current="page"
      class="active"
      href="/"
    >
      home 
    </a>
    <a
      href="/admin"
    >
       admin
    </a>
  </header>
  <form>
    <p />
    <label>
      name
      <input
        placeholder="name here"
        type="text"
        value=""
      />
    </label>
    <label>
      age
      <input
        placeholder="age"
        type="number"
        value=""
      />
    </label>
    <label>
      bio
      <input
        placeholder="bio"
        type="text"
        value=""
      />
    </label>
    <label>
      image
      <input
        type="text"
        value=""
      />
    </label>
    <label>
      breed
      <input
        placeholder="breed"
        type="text"
        value=""
      />
    </label>
    <button>
      save
    </button>
  </form>
</div>
`;
